README.md

```diff
@@ -62,16 +62,17 @@ $ kubectl apply -f deploy/upstream
 
 #### Installing an operator using Marketplace
 
-The following section assumes that Marketplace was installed in the `marketplace` namespace. For Marketplace to function you need to have at least one `OperatorSource` CR present on the cluster. To get started you can use the `OperatorSource` for [upstream-community-operators](deploy/examples/upstream.operatorsource.cr.yaml). If you are on an OKD cluster, you can skip this step as the `OperatorSource` for [community-operators](deploy/examples/community.operatorsource.cr.yaml) is installed by default instead.
+The following section assumes that Marketplace was installed in the `marketplace` namespace. For Marketplace to function you need to have at least one `OperatorSource` CR present on the cluster. To get started you can use the `OperatorSource` for [upstream-community-operators]. If you are on an OKD cluster, you can skip this step as the `OperatorSource` for [community-operators] is installed by default instead.
 ```bash
-$ kubectl apply -f deploy/examples/upstream.operatorsource.cr.yaml
+$ kubectl apply -f deploy/examples/07_upstream_operatorsource.cr.yaml
```

Comment:
This file does not exist, it looks like it should point to `deploy/upstream/07_upstream_operatorsource.cr.yaml`

Reply:
Fixed